src/popover.coffee

```diff
@@ -234,8 +234,15 @@ Ember.Widgets.PopoverComponent.reopenClass
   rootElement: '.ember-application'
   hideAll: -> $(document).trigger('popover:hide')
 
-  popup: (options) ->
-    @hideAll()
+  ###*
+   * Shows a popup with options used for PopoverComponent.
+   * @param {Object} options All options used for PopoverComponent.
+   * @param {boolean} [hideOthers=true] Indicates if other popover should be hidden when a new one
+   *    is shown. By default, it's set to true.
+  ###
+  popup: (options, hideOthers = true) ->
```

Comment:
And hideOthers should be part of the attribute of popover component, similar to the rootElement. And you can have the default value there. But you may want to do something like this:
```
  popup: (options) ->
    if Ember.isNone(options)
      hideOthers = @hideOthers
      rootElement = @rootElement
    else
      hideOthers = options.hideOthers or @hideOthers
      rootElement = options.rootElement or @rootElement

    if hideOthers
      @hideAll()
    popover = this.create options
    if popover.get('targetObject.container')
      popover.set 'container', popover.get('targetObject.container')
    popover.appendTo rootElement
    popover
```